.gitignore

```diff
@@ -6,5 +6,4 @@ coverage.out
 .env.local
 recipe.db
 bin
-go.work.sum
-fuego
\ No newline at end of file
+go.work.sum
```

Comment:
I'm afraid changing this might be problematic if you try to build fuego cli, I think it's ok to have this ignore. Unless it prevents something else to be tracked?

Reply:
it prevents any files in cmd/fuego to be tracked